### PR TITLE
Expect-CT and CT in HTTP Pinning

### PR DESCRIPTION
**Pull request by ThunderSon**
HPKP should no longer be used for HTTP pinning.

It has been replaced by Certificate Transparency coupled with the `Expect-CT` header.

### CODE REVIEW

**Comment by ThunderSon** [comment]
@commjoen care to give this a quick review? 😊

File: pages/controls/Certificate_and_Public_Key_Pinning.md [hunk 1]
```diff
@@ -315,25 +315,11 @@ Java, iOS, .Net, and OpenSSL.
...
-Please note that [RFC 7469](http://www.rfc-editor.org/rfc/rfc7469.txt)
-is controversial since it allows overrides for locally installed
-authorities. That is, it allows an adversary or other party who
-successfully phishes the user to override a known good pinset with
-non-authentic or fraudulent information. Second, the reporting mechanism
-is suppressed from broken pinsets, so a complying user agent will be
-complicit in the cover up after the fact. That is, the reporting of the
-broken pinset is called out as **MUST NOT** report
-[\[1](https://en.wikipedia.org/w/index.php?title=HTTP_Public_Key_Pinning)\].
+[Expect-CT](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expect-CT) header allows sites to opt in to the [Certificate Transparency](https://developer.mozilla.org/en-US/docs/Web/Security/Certificate_Transparency) framework, in report or enforcement mode, based on the readiness of the application.
```
Comment (commjoen) [inline pages/controls/Certificate_and_Public_Key_Pinning.md:318]: Good additions! To make it more complete: Maybe it is good to add a little bridge here: e.g. that http pinning (with its header) was where it started, but with the flaws found in the implementation ending up in the deprecation and removal of HPKP support in major browsers and that now expect-ct is the new kid?

Comment (ThunderSon) [reply pages/controls/Certificate_and_Public_Key_Pinning.md:318]: Noted, I will send in an update accordingly.

Comment (ThunderSon) [reply pages/controls/Certificate_and_Public_Key_Pinning.md:318]: Added a note on HPKP with it being replaced by CT and the header. Let me know what you think about it.

**Comment by commjoen** [comment]
Thanks!

**Pull request merged**